Escape dashes in verbatim text to have groff render them as-is rather
than as \x{2010}.

diff -Naur --exclude=debian perl-5.8.8.orig/lib/Pod/Man.pm perl-5.8.8/lib/Pod/Man.pm
--- perl-5.8.8.orig/lib/Pod/Man.pm	2003-03-31 21:16:36.000000000 +1000
+++ perl-5.8.8/lib/Pod/Man.pm	2006-02-01 23:39:27.000000000 +1100
@@ -517,6 +517,7 @@
     my $lines = tr/\n/\n/;
     1 while s/^(.*?)(\t+)/$1 . ' ' x (length ($2) * 8 - length ($1) % 8)/me;
     s/\\/\\e/g;
+    s/-/\\-/g;
     s/^(\s*\S)/'\&' . $1/gme;
     $self->makespace;
     $self->output (".Vb $lines\n$_.Ve\n");
diff -Naur --exclude=debian perl-5.8.8.orig/lib/Pod/t/basic.man perl-5.8.8/lib/Pod/t/basic.man
--- perl-5.8.8.orig/lib/Pod/t/basic.man	2002-01-03 01:33:15.000000000 +1100
+++ perl-5.8.8/lib/Pod/t/basic.man	2006-02-01 23:39:27.000000000 +1100
@@ -332,7 +332,7 @@
 .PP
 .Vb 2
 \& This is.  And this:    is an internal tab.  It should be:
-\&                    |--| <= lined up with that.
+\&                    |\-\-| <= lined up with that.
 .Ve
 .PP
 (Tricky, but tabs should be expanded before the translator starts in on
